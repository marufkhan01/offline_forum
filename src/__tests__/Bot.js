import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import Bot from '../components/Bot/Bot';

it('Button component should render as expected',() =>{
    const component = shallow(<Bot />);
    const tree = toJson(component);
    expect(tree).toMatchSnapshot();
});