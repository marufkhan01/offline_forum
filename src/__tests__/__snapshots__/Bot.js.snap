// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button component should render as expected 1`] = `
<div
  className="mx-auto w-1/2 h-auto mb-4 mt-8 pt-8"
>
  <div
    className="mx-auto h-64 mb-4 p-4 border flex flex-col justify-end rounded shadow border overflow-y-scroll"
  >
    <TypingIndicator
      typing={false}
    />
  </div>
  <MessageForm
    onSubmit={[Function]}
  />
</div>
`;
